fdbclient/BackupAgentBase.actor.cpp

```diff
@@ -333,23 +353,38 @@ void decodeBackupLogValue(Arena& arena,
 								logValue.param1 = logValue.param1.withPrefix(addPrefix, tempArena);
 								logValue.param2 = logValue.param2.withPrefix(addPrefix, tempArena);
 							}
-							result.push_back_deep(arena, logValue);
-							mutationSize += logValue.expectedSize();
+							result->push_back_deep(*arena, logValue);
+							*mutationSize += logValue.expectedSize();
+						}
+						if (originalLogValue.param1 == logValue.param1 && originalLogValue.param2 == logValue.param2) {
```

Comment:
Given the usecase we want to support is the following:

Add/remove prefix should not update "encryption domain" (including ClearRange), should we be doing the find the originalMutation encryptionDomain and make sure if matches with mutation encryption domain after add/remove prefix and if not, throw an error? (I thought throwing error was the agreed upon plan..)

Reply:
Discussed offline and for right now we will allow changing tenants by adding/removing prefixes

Reply:
what's the blocker for throwing an error to disallow changing tenants, and what's the future plan?

Reply:
Please add code documentation describing the usecases handled?